Assets/_BForBoss/Scripts/UserInterface/GameplaySettingsViewBehaviour.cs

```diff
@@ -0,0 +1,134 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BForBoss
+{
+    public class GameplaySettingsViewBehaviour : MonoBehaviour
+    {
+        [SerializeField] private TMP_Dropdown _povDropdown = null;
+        [SerializeField] private Toggle _showFPSToggle = null;
+        [SerializeField] private Toggle _showRAMToggle = null;
+        [SerializeField] private Toggle _showPCSpecsToggle = null;
+
+        private GameplaySettingsViewModel _viewModel = null;
+        
+        public void Initialize(IThirdPerson thirdPersonSettings)
+        {
+            _viewModel = new GameplaySettingsViewModel(thirdPersonSettings);
+            SetViews();
+            BindModel();
+            _viewModel.SetGraphy();
+#if !DEVELOPMENT_BUILD && !UNITY_EDITOR
+            HideSettingsForProduction();
+#endif
+        }
+
+        private void BindModel()
+        {
+            _showFPSToggle.onValueChanged.RemoveAllListeners();
+            _showFPSToggle.onValueChanged.AddListener(isOn =>
+            {
+                _viewModel.SetShowFPS(isOn);
+            });
+            
+            _showRAMToggle.onValueChanged.RemoveAllListeners();
+            _showRAMToggle.onValueChanged.AddListener(isOn =>
+            {
+                _viewModel.SetShowRAMUsage(isOn);
+            });
+            
+            _showPCSpecsToggle.onValueChanged.RemoveAllListeners();
+            _showPCSpecsToggle.onValueChanged.AddListener(isOn =>
+            {
+                _viewModel.SetShowPCSpecifications(isOn);
+            });
+            
+            _povDropdown.onValueChanged.RemoveAllListeners();
+            _povDropdown.onValueChanged.AddListener(value =>
+            {
+                _viewModel.SetPOV(value);
+            });
+        }
+
+        private void SetViews()
+        {
+            _showFPSToggle.isOn = _viewModel.IsShowingFPS;
+            _showPCSpecsToggle.isOn = _viewModel.IsShowingPCSpecs;
+            _showRAMToggle.isOn = _viewModel.IsShowingRAM;
+            _povDropdown.value = _viewModel.IsThirdPersonView ? 1 : 0;
+        }
+
+        private void HideSettingsForProduction()
+        {
+            _viewModel.SetShowPCSpecifications(false);
+            _viewModel.SetShowRAMUsage(false);
+            _showPCSpecsToggle.gameObject.SetActive(false);
+            _showRAMToggle.gameObject.SetActive(false);
+        }
+    }
+    
+    public class GameplaySettingsViewModel
+    {
+        private struct PlayerPrefKey
+        {
+            public const string ShowFPS = "ShowFPS";
+            public const string ShowRAMUsage = "ShowRAMUsage";
+            public const string ShowPCSpecs = "ShowPCSpecs";
+        }
```

Comment:
Can you make this public? Otherwise the player pref tool can't access it

Reply:
I'll be moving the player prefs in another ticket.  I'll do that then